### PR TITLE
Enable SASS variables in JS

### PR DESCRIPTION
**Pull request by heldergoncalves92**
**Goal:** Enable to share stylish values from CSS to JS.
**Outcome:** 
- A single source of truth between CSS and JS;
- Cheaper to maintain;
- Enable themes for JS components, i.e., Monaco Editor;

### CODE REVIEW

File: Example/ExampleView/ExampleView.scss [hunk 1]
```diff
@@ -1,4 +1,10 @@
-﻿.wrapper {
+﻿$someVar: "Some Pretty Value!";
+
+:export {
+    exportedVarible: $someVar;
```
Comment (alvesmiguel1) [inline Example/ExampleView/ExampleView.scss]: exportedVariable

Comment (heldergoncalves92) [reply Example/ExampleView/ExampleView.scss]: Fixed!

File: ViewGenerator/ViewGenerator.nuspec [hunk 2]
```diff
@@ -2,14 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>ViewGenerator</id>
-    <version>1.0.290</version>
+    <version>1.0.293</version>
```
Comment (tbragaf) [inline ViewGenerator/ViewGenerator.nuspec]: why 3 versions ahead?

Comment (heldergoncalves92) [reply ViewGenerator/ViewGenerator.nuspec]: I just incremented every time I needed to raise a version during development. Is that a problem?

Comment (tbragaf) [reply ViewGenerator/ViewGenerator.nuspec]: there's no harm to it
but you just need it once to update it in SS

Comment (alvesmiguel1) [reply ViewGenerator/ViewGenerator.nuspec]: It's ok, we usually do that as well during development

**Pull request merged**